### ISSUE TITLE
Duplicate identifier 'svgContent'

### ISSUE THREAD

**Issue by jmcmullen** [fabriciomendonca/stencil-inline-svg#2]
I am having trouble getting this to work in my project. After following the read me file I encounter this error:

```
Duplicate identifier 'svgContent'.ts(2300)
images.d.ts(4, 18): 'svgContent' was also declared here.
```

The conflicting file is part of stencil: `node_modules/@stencil/core/internal/images.d.ts`

```
declare module '*.svg' {
  const src: string;
  export default src;
}
```

"@stencil/core": "1.4.0",
"stencil-inline-svg": "0.0.2"

Any suggestions?

**Comment by fabriciomendonca** [comment]
Stenciljs on version 1.0 added the types for svg images. You don't need to create the file `types.d.ts` anymore. I will update the README file.

[EDIT] It seems that there is a problem with version 1.0: it is converting the svg content into a base64 string. I will try to find a solution for this once it is a blocker for some people using stenciljs: https://github.com/ionic-team/stencil/issues/1738

**Comment by fabriciomendonca** [comment]
I have just released the version 1.0.0 of the package, making it work with Stencil ^1.4.0.
It will check if the source text is a rollup imagePlugin export string, extract the base64 string, decode it and return a new export for the inline svg.
